assets/src/edit-story/karma/_init.js

```diff
@@ -0,0 +1,39 @@
+
+let rootEl;
+
+beforeEach(() => {
+  rootEl = document.createElement('test-root');
+  rootEl.id = 'test-root-' + Math.floor(Math.random() * 10000000);
```

Comment:
What purpose does Math.random serve?

Reply:
Not needed. Just something I was debugging from the puppeteer side, but no longer an issue.